### ISSUE TITLE
Proposal: Alphabetic ordering when doing multiple property destructing assignment

### ISSUE THREAD

**Issue by gaizka-allende** [eslint/eslint#8076]
Enforce alphabetic ordering when doing multiple property destructing assignment;

[x ] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

```js
const o = {p: 42, q: true};
const {p, q} = o; //valid
const {q, p} = o; //invalid
```

**Why should this rule be included in ESLint (instead of a plugin)?**
Not sure

**Comment by vitorbal** [comment]
@gaizka-allende to be clear, are you proposing a new rule or suggesting an improvement to another existing rule?
I'm thinking that this proposal might fit better as an enhancement for http://eslint.org/docs/rules/sort-vars (or for http://eslint.org/docs/rules/sort-keys, not sure).

**Comment by gaizka-allende** [comment]
ok. cool, it's already implemented
